contracts/farm-bot.sol

```diff
@@ -71,9 +81,8 @@ contract FarmBot is Owned, FarmbotERC20 {
         revoBounty = IRevoBounty(_revoBounty);
     }
 
-    function updatePaths(address[] calldata _path0, address[] calldata _path1) external onlyOwner {
-        path0 = _path0;
-        path1 = _path1;
+    function updatePaths(address[][2][] memory _paths) external onlyOwner {
+	paths = _paths;
```

Comment:
should there be a check here to make sure the rewards tokens are still at the ends of the new paths?

Reply:
otherwise the owner could force farm bots to buy an unrelated token, and drive up its price or something. kind of a weird threat model admittedly but easy enough to prevent

Reply:
Yeah, that's probably a good idea, although that check is not super trivial to implement. It's not obvious, but the `claimRewards` call where the swapping occurs will actually fail if the end tokens in the path don't align with the constituent tokens of the LP, since we won't have as much of `token0`/`token1` as we say we do when calling the router's `addLiquidity` method. (I'm _pretty_ sure this is true, not 100% though.)

Reply:
#11